create-function-deps_restore_dependencies.sql

```diff
@@ -0,0 +1,31 @@
+CREATE OR REPLACE FUNCTION public.deps_restore_dependencies(p_view_schema IN VARCHAR, p_view_name IN VARCHAR)
+RETURNS NULL
```

Comment:
is this supposed to be `void` ? Its not recognizing NULL as an optio. Or is it supposed to be `RETURNS NULL ON NULL INPUT` ?

Reply:
right, void. I had to change that on the other one during testing.